eth-connector/src/fee.rs

```diff
@@ -0,0 +1,28 @@
+use near_sdk::{
+    borsh::{self, BorshDeserialize, BorshSerialize},
+    collections::UnorderedMap,
+    json_types::U128,
+    serde::{Deserialize, Serialize},
+    AccountId,
+};
+
+#[derive(Copy, Debug, Serialize, Deserialize, Clone)]
+pub enum FeeType {
+    Deposit,
+    Withdraw,
+}
+
+#[derive(BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, Copy)]
+pub struct Fee {
+    pub fee_percentage: U128,
```

Comment:
In this particular implementation, the fee is not a percentage but rather a coefficient of a numerator, while the denominator is equal to `1_000_000` due to 6-decimals precision. So, perhaps, the more proper naming would be `fee_numerator`? WDYT?

CC @karim-en @aleksuss @mrLSD

Reply:
PS: Actually the `precision` could be also part of this fee config instead of being hardcoded.

Reply:
@karim-en it shouldn't be `U128`. Just `u32`.

Reply:
9939668b661663eb9d830b3ab65ba2ea0596b6ed